### ISSUE TITLE
Flapjack blog category minicons

### ISSUE THREAD

**Issue by elizbond** [cfpb/design-manual#130]
I'm not sure where we left off on the Flapjack minicons since this branding conversation happened before Github. @Dnpizarro and @ohsk were working on it at the time. Can you both provide your latest files so we can wrap up this conversation?

Consumer Information ( @duelj is this the same as Consumer Finance?)
Blog
Regulations 
Engagements and Events
Announcements and Updates (Press release)
Innovation
Share

**Comment by elizbond** [comment]
@Dnpizarro and @ohsk reminder

**Comment by juidai** [comment]
Sorry, but this list needs to be updated. Here's the icons we need:
- share
- blog
- consumer information (blogs that is consumer centric i.e. paying for college, tips on purchasing a home)
- engagements & events (town halls and stuff where cfpb shows up)
- announcements & updates (regulation type of news or issues the agency will try to tackle)
- innovation (blog posts around e-regs, hmda, or anything that is unique and interesting cfpb is doing that is out of the box innovative stuff)

**Comment by Dnpizarro** [comment]
Below are the updates from where we last left off. Announcements/Press Release has not been decided, nor has Consumer finance. The only one that not on the list is "Consumer information", this is new, however this could be referring to "Consumer finance." @juidai are these the same or does there need to be another icon developed?

![flapjack_minicons](https://cloud.githubusercontent.com/assets/6852363/2791475/4ac9c7ec-cbc7-11e3-9823-206f76ae869e.png)

**Comment by ohsk** [comment]
@Dnpizzarro, two thoughts:
- Blog: is that the latest edit pencil? I thought we had one with a curves tip. We. Might want to exaggerate its scale a bit as well, relative to the speech bubble, for scalability...
- Innovation: could we try removing one of the lines around the base of the bulb? Seems a little busy down there.

**Comment by Dnpizarro** [comment]
I also developed something for regulations but not sure if its still needed.
![regulations](https://cloud.githubusercontent.com/assets/6852363/2791568/1db804ac-cbc8-11e3-8ae8-ff560c145c70.png)

**Comment by Dnpizarro** [comment]
Here is the update for innovation, based on @ohsk feedback:

![innovation](https://cloud.githubusercontent.com/assets/6852363/2791724/b81e8de4-cbc9-11e3-8bd3-553baf07b990.png)

**Comment by juidai** [comment]
consumer information is about consumer financial stuff. it is the same even though it sounds different. i am not sure if money is a good representation of consumer financial advice. 

otherwise, all the other icons look great. thanks.

**Comment by duelj** [comment]
Here were some concepts I had put together a while back for "Consumer Finance" maybe they will help with some ideas for the "Consumer Information" icon. 
![screen shot 2014-04-24 at 1 24 01 pm](https://cloud.githubusercontent.com/assets/6873734/2792626/751bde8c-cbd5-11e3-9f76-c9ad19215dee.png)

**Comment by juidai** [comment]
i like the 2 piggie bank. piggie banks indicate consumer/saving/personal finance. $ might be too hard to read when shrunken down but the circle looks like coins.

**Comment by Dnpizarro** [comment]
Here are some revisions for the blog minicon.

![blog](https://cloud.githubusercontent.com/assets/6852363/2792806/9dff5458-cbd7-11e3-8c2a-dd8fecb240e4.png)

**Comment by Dnpizarro** [comment]
Out of the set that @duelj suggested for "Consumer finance" I think the wallet with the bill might work better. The piggy I feel is too heavily geared towards savings. I would steer away front he eye symbolism, feels a bit like the freemasons. The dollar, I would agree with you in that it lacks any real meaning.

**Comment by elizbond** [comment]
I agree with @Dnpizarro on the wallet, I'm not 100% sure it's the best representation though. There might be other options out there to explore. The piggy bank does seem really heavy and I'm concerned about the level of detail at the smaller level.

**Comment by duelj** [comment]
@Dnpizarro I think it is important to remember that the icon is now for Consumer Information rather than Consumer Finance. Either way it is a complex subject matter and I believe whatever icon we land on will only represent a portion of it. If the piggy bank leans too far towards savings perhaps the wallet leans too far towards spending and is not representative of those who don't carry wallets. Additionally, neither of those icons (piggy bank or wallet) take credit cards and the like into account, which is a big piece of consumer finance and information that we provide. 

I think we probably need to do some more exploration to find the right fit but I fear that there will always be an argument that the icon doesn't fully represent the diverse subject matter, and bearing that in mind might help us find a more effective representation.

**Comment by nataliafitzgerald** [comment]
This is the megaphone version I referenced in the Design Crit. 

![screen shot 2014-04-24 at 2 50 26 pm](https://cloud.githubusercontent.com/assets/6887128/2793570/56790d90-cbe1-11e3-9e75-2031ae82ded6.png)

**Comment by Dnpizarro** [comment]
Below are a couple more versions to the announcements minicon. @nataliafitzgerald  I do remember discussing the megaphone option you mentioned during the crit. My only concern with that version is that it associates to cheerleading as well. We also had some other versions where we took the sound symbolism off of the megaphone. Do any of these options resonate with anyone?

![announcements](https://cloud.githubusercontent.com/assets/6852363/2794116/ecddd4b2-cbe8-11e3-9653-3228ce6e848e.png)

**Comment by Dnpizarro** [comment]
I developed a couple more options for the blog minicon based on suggestions from the crit. These options are less consumer & financed focused, and more geared towards information. Let me know if you feel these are appropriate or oversimplified for the content. 

![consumerinfo](https://cloud.githubusercontent.com/assets/6852363/2794203/0459b9c0-cbea-11e3-88ec-8ee1c81c7003.png)

**Comment by juidai** [comment]
events should be less focused on calendar and more about interaction/engagement with consumers. team brainstorming came up with microphone.

**Comment by ell0ell0** [comment]
so for the " i " spend some time refining the letterform. I think the bottom rectangle of the i letter should be squared instead of rounded. That should help create enough contrast between the dot on the bottom of the letter that it'll read easier at smaller sizes. 

The reversed out version should just be the " i " letter on black with no white stroke circle around it. Putting a circle in a circle is a little weird and just makes the " i " smaller and then harder to read at small sizes.

I threw this together super quick, so you should refine it more, but if the i is shorter and stubier than the traditional letterform it also starts to resemble a super simplified and abstracted person.

![i](https://cloud.githubusercontent.com/assets/3589375/2794637/2ed92474-cbef-11e3-837a-81db88e3de3e.png)

**Comment by Dnpizarro** [comment]
Per @juidai comments on the event minicon, I came up with some microphone options:

![events](https://cloud.githubusercontent.com/assets/6852363/2795246/c09bdbba-cbf7-11e3-8d6f-7ac845ef7d99.png)

I see where the intentions are to embody interaction/engagement, which I'm actually for, however my only concern is that the calendar tends to be the conventional and more recognizable option. It would be worthwhile to do some user testing with this set once we finalize everything.

**Comment by Dnpizarro** [comment]
I took into account some of @ell0ell0 suggestions and made some revisions to the "Consumer information" minicon. I'd agree that the previous version was too rounded, however we don't use  sharp edges in the designs of our minicons. I gave the two versions below a slight rounded edge to maintain a consistent treatment with our set and to give it the needed contrast. 

![info](https://cloud.githubusercontent.com/assets/6852363/2801394/83e8d10c-cc81-11e3-9e40-6fcd70ac3dd2.png)

**Comment by elizbond** [comment]
I agree with @Dnpizarro on the microphone/calendar dilemma. I understand the intention but the calendar is a more recognizable symbol for events. Also, if we go with the megaphone option for announcements, the symbols are too closely related and could lead to confusion.

According to my notes, here is where we are so far---

share - done
![4ac9c7ec-cbc7-11e3-9823-206f76ae869e](https://cloud.githubusercontent.com/assets/6798337/2802103/9479de0e-cc8a-11e3-80a7-ad62a72d1562.png)

blog - still to be decided, I prefer # 1
![9dff5458-cbd7-11e3-8c2a-dd8fecb240e4](https://cloud.githubusercontent.com/assets/6798337/2802052/02a873a0-cc8a-11e3-8871-11b9e027935e.png)

consumer information - other thoughts or suggestions?
![83e8d10c-cc81-11e3-9e40-6fcd70ac3dd2](https://cloud.githubusercontent.com/assets/6798337/2802058/1b8bc37c-cc8a-11e3-8445-c49be0f23735.png)

engagements & events - calendar vs microphone, still to be decided

announcements & updates - still to be decided, I prefer # 4
![ecddd4b2-cbe8-11e3-9653-3228ce6e848e](https://cloud.githubusercontent.com/assets/6798337/2802076/4b147cd8-cc8a-11e3-8374-f8c9484cd901.png)

innovation - done
![b81e8de4-cbc9-11e3-8bd3-553baf07b990](https://cloud.githubusercontent.com/assets/6798337/2802092/799e0024-cc8a-11e3-8e0a-f935a9f7b3b4.png)

**Comment by ohsk** [comment]
A few thoughts:

### Blog

I prefer the composition of # 1, but I think we need to rethink the in-circle version. These should be less outline, more fill-based. For example, the way both of the credit card and other items are filled in this example: 
![image](https://cloud.githubusercontent.com/assets/1689222/2803839/b5036f48-cca1-11e3-9f77-a2bc0f914e06.png) ![image](https://cloud.githubusercontent.com/assets/1689222/2803849/e9eb2b42-cca1-11e3-9103-a73a56357f6b.png)

### Consumer information

Preferring # 1 with the thinner "i" proportion, but maybe add a little whitespace between the dot and the line?

### Engagements & Events

I'm down with either option for this. I do think we could include both of the approaches in the library either way, as I think they'd eventually be useful elsewhere.
![image](https://cloud.githubusercontent.com/assets/1689222/2803998/161b710c-cca4-11e3-89d8-91fe2b0217c7.png)  
![image](https://cloud.githubusercontent.com/assets/1689222/2804007/2b1272d6-cca4-11e3-8bcc-f70140902c2b.png)

### Announcements & Updates

I prefer # 3 or # 4 generally. I did try some small tweaks to bring more in line with our style, as I think it should be just slightly more geometric, with less curvature to the front of the megaphone.

![image](https://cloud.githubusercontent.com/assets/1689222/2804012/434fd712-cca4-11e3-9511-2ffc3ccc1ef2.png)

### Innovation

Can we also see what it looks like with the lightbulb filled-in in the circle version?

**Comment by Dnpizarro** [comment]
I decided to compile all of the minicons, both decided and undecided, so we call all see the full set. I included into these revisions @ohsk most recent suggestions. This set is also scaled down to give a more accurate view of the minicons at a smaller scale. 

The majority of the minicons are still undecided, however we are getting closer to a resolution. 
**Share:** DONE.
**Consumer Information:** DONE, unless there are another suggestions.
**Innovation:** Filled or outline?
**Events:** I like the suggestion to keep both in the library. With that said we still need to select one microphone option if we chose to include it into the set.
**Announcements:** Filled or outlined? Curved front part of cone or straight?
**Blog:** Filled bubble or outlined?

If you have any other suggestions outside of what I mentioned, please feel free to comment.

![full_set](https://cloud.githubusercontent.com/assets/6852363/2805481/c55303e6-ccb9-11e3-8a09-beabc4e060a5.png)

**Comment by duelj** [comment]
Hey all, great work on this! We are very close, we discussed the options in design review today and here is where things stand from Flapjack: 

Consumer Information : 
![screen shot 2014-04-25 at 4 42 24 pm](https://cloud.githubusercontent.com/assets/6873734/2805513/2fd94608-ccba-11e3-9278-e3134a8c5c48.png)
@Dnpizarro  #1 - with the tweaks that @ohsk mentioned.

Blog: #1 But I was also curious about seeing the pencil black filled with white outlines and just the bubble full white fill. Thought the balance might be nice.
![screen shot 2014-04-25 at 4 41 45 pm](https://cloud.githubusercontent.com/assets/6873734/2805516/3a6929ee-ccba-11e3-84d1-cb205671d190.png)

Engagements & events: #4 - keeping the calendar also makes sense but for our purposes will need the microphone. 
![screen shot 2014-04-25 at 4 42 12 pm](https://cloud.githubusercontent.com/assets/6873734/2805526/667b79ba-ccba-11e3-936a-5243b0d8648c.png)

Announcements: @ohsk #5 
![screen shot 2014-04-25 at 4 42 05 pm](https://cloud.githubusercontent.com/assets/6873734/2805530/7195300c-ccba-11e3-94f9-f74e63434b5c.png)

**Comment by Dnpizarro** [comment]
I made another round of revisions, which include @duelj suggestions. We're half way there, still 3 more to finalize. Please make your suggestions on the remaining minicons: **Innovation, Blog, and Announcements**

![full_set2](https://cloud.githubusercontent.com/assets/6852363/2806005/55fee282-ccc1-11e3-9bbb-ce24922c28ae.png)

**Comment by Dnpizarro** [comment]
Here is a preview of the set in context.

![sample_set](https://cloud.githubusercontent.com/assets/6852363/2806042/e0f08b70-ccc1-11e3-8c9c-4e6ae32ff38d.png)

**Comment by Dnpizarro** [comment]
**One note:** The minicons will not appear blurry once they are integrated into the minicon font. With that said, I am noticing that the innovation minicon is losing its threading at the base of the shape. This could be of course due to my retina display. In any case, this could be tweaked further later once I begin integrating the set into the font.

**Comment by benguhin** [comment]
Looks great! Two notes that you can feel free to ignore:
1. I think that the blog icon reads a lot like "comment," which is kinda accurate but not really
2. Compared to the iOS version, the arrow in the share icon looks pretty flimsy, with more emphasis on the box than the arrow. Beef it up?

**Comment by ohsk** [comment]
@benguhin, I agree with both your comments. Headed out of town, but just for posterity's sake, here's some older versions for both **Blog** and **Share**.

![image](https://cloud.githubusercontent.com/assets/1689222/2810416/cdeef2e6-cdbb-11e3-90a7-dd833acec4dd.png)

![image](https://cloud.githubusercontent.com/assets/1689222/2810417/e3d43828-cdbb-11e3-8164-edd6f71ea1c3.png)

**Comment by Dnpizarro** [comment]
@benguhin, has a point with the share icon. I added another older version next to the one @ohsk  suggested. I think back when we were looking at the iOS option, we debated whether the "gene effect" of the arrow aligned with how we were designing our minicons. With that said, if we go with the iOS version of share, are we ok with making that exception for this minicon?

![share](https://cloud.githubusercontent.com/assets/6852363/2819947/5eee6842-cef4-11e3-83fd-971cfb366e62.png)

**Comment by nataliafitzgerald** [comment]
For the "Engagements & Events" the lines on the microphone are getting lost at actual minicon size. For the sake of simplicity when scaled I would recommend version #1 below. 

![161b710c-cca4-11e3-89d8-91fe2b0217c7](https://cloud.githubusercontent.com/assets/6887128/2820692/2c086b2c-cefd-11e3-8e63-2401bb57533e.png)

**Comment by Scotchester** [comment]
@Dnpizarro #1 is the external link minicon.

**Comment by Dnpizarro** [comment]
@Scotchester its not exactly the external link minicon, but to your point it is too closely related so that rules out option 1. I think @nataliafitzgerald could be right about the scaling issue, however keep in mind that the minicons will look sharper once in the font format. I can include versions 1 and 4 of the microphone into the font as part of our initial test and later remove the one we decide not to use.

**Comment by nataliafitzgerald** [comment]
I agree with @elizbond and @Dnpizarro's concerns about using the microphone for "Engagements & Events." An additional concern I have with the microphone is that implies communication that is directed outward. If we are talking primarily about field hearings, these hearings are a unique forum where consumers can speak directly with the CFPB and make their experiences heard. The heart of the field hearings is when consumers have the opportunity to speak out. I imagine that you could argue that the microphone could suggest that you are giving a voice to the consumer but given that this is a government website, I think the more likely interpretation would be that the government is doing the speaking. @juidai could you provide us with a bit of background on how the team landed on the microphone? Maybe if you walk us through it a bit we can better understand the thinking behind it.

**Comment by ell0ell0** [comment]
My one comment strictly speaking is the counterforms (white space) all need to be opened up. i.e. the space btween the dot and the vertical line in the "i" of consumer information, the space int he handle of the megaphone for recent press, space between microphone and stand for engagement and events, etc. When these icons get small which is by default the minicon use case (16pt text) those spaces start to disappear, especially on a non retina display.

**Comment by juidai** [comment]
"Engagements and Events" will comprise of blog post announcing - CFPB's field hearings (http://www.consumerfinance.gov/blog/live-from-washington-d-c-2/), ask users to engage us about the rules we are in the process of creating, RFI (request for information) on a topic such as student credit cards (http://files.consumerfinance.gov/f/201311_cfpb_letter-to-student-loan-servicers.pdf), Know before you owe - consumer feedback on mortgage forms (http://www.consumerfinance.gov/blog/prepaid-cards-help-design-a-new-disclosure/), etc. It's a forum to ask consumer and industry folks to engage with us. That's the reason we moved away from calendar since calendar is associated with a time and sometimes a place. The other forums I listed are not time based but rather asking the public to speak out. We like microphone because it indicates voice or speaking into something about something. I do not think microphone is outward, Siri uses microphone, and it's very recognized as a way to speak into a microphone.

**Comment by elizbond** [comment]
I'm still not sold on the microphone. If the label is "Engagement and Events" it's signaling an arrangement to do something or go somewhere at a fixed time, not engage or interact with the CFPB. If the icon is a microphone, would it make more sense for the category to be "speak out" or something similar?

I understand the appeal of the microphone because it indicates voicing your opinion but it's point of view doesn't really fit with the other minicons, which I think is what @nataliafitzgerald is getting at. The other icons in this library are _from_ the CFPB. For example, "announcements" is a megaphone to amplify news from the CFPB. Or you're sharing information from the CFPB. It's natural to interpret the microphone the same way. Since these are working as a library, they should have the same point of view. It's because of this that I worry the microphone and the megaphone could be easily confused. I feel like the icon for "engagement and events" should be something receptive, since the CFPB is receiving the opinions of consumers and the industry. 

Also, I'm concerned about the multiple libraries being used across the site with #166. Should we consider simplifying the icons being used? Should each icon be specific or can we overlap? For example the megaphone was originally created for press releases, could we use it for both announcements and press releases?

**Comment by elizbond** [comment]
@juidai @duelj @ell0ell0 How many minicon sets will be needed throughout the site? 
1. Blog
2. Newsroom
3. ...? Anywhere else?

**Comment by juidai** [comment]
We have not taken an inventory of the whole site to figure out how many minicon sets we will need. As we design/build each section and as we come across illustrations or minicons, we just ask Design Ops for help.

**Comment by juidai** [comment]
@elizbond - you raised a good question about minicon library sets. I believe library management and usage should be up to Design Ops to decide. My concern is creating legible icons that convey the intention the icon is trying to describe on a specific section. If you guys need some more time to sort the direction of the minicons because our usage is beyond the original expectation, that's cool. I respect your need to rethink how to use the minicons since it has grown beyond the initial intention. I just need a timeline on resolution so I can pass it along to my product owner and stakeholder. Thanks.

**Comment by elizbond** [comment]
@juidai I think in order for us to understand the full scope and provide a timeline we need to know how many possible sets and how many possible icons might exist. That way we can of these libraries as a full language. I agree that the priority should be creating legible icons that convey the intention of the content, so I'm concerned we might be getting too specific or not specific enough on some icons. Knowing the full list of possible icons would help understand the intention of each icon and set.

**Comment by Dnpizarro** [comment]
I would echo @elizbond point. We can give you a better assessment of our timeline once we know if there are other sets Flapjack has considered that have not been mentioned in this thread. Of course to your point @juidai, the need for minicons comes as you design/build, but moving forward knowing 
what your full needs are currently, regarding minicons, would helps us better resolve some of the issues we've been discussing.

**Comment by ell0ell0** [comment]
@elizbond @Dnpizarro @nataliafitzgerald  Guys in all seriousness we can't tell you our exact needs. Yes there will be more sets in the future... How many? We'll know when the site is finished... Can we wait till then? Of course not... 

Lets stop playing chicken and the egg and going in circles and focus on accomplishing something.

Ignore the fact that there will be more icons. let's take this one step at a time.

What do you need right now to finish the few you all are working on?

**Comment by juidai** [comment]
I understand designers sometimes want to know the complete requirements before crafting the designs. But we are working in an agile process and that means we define requirements when we get to the problem. I understand that would be a lot easier to build but it's not very efficient for software development. Is there any way to evaluate the minicon creation process to be more inline with the rest of T&I process?

**Comment by Dnpizarro** [comment]
@ell0ell0 if anything we were asking for clarification to better understand your needs, and if along the process we raise questions its just part of trying to arrive at a solution that makes most sense. 

@juidai , I agree with your point about agile. I think we need to stay flexible through this process, so what I would suggest at this point is to keep our options open as the set evolves. So, including both calendar and microphone into the set would be useful as both you and @elizbond have valid points. Moving forward, so as to not get stuck on any one minicon we should add any potentially useful minicons into our library incase we all decide that the alternate would work better.

**Comment by duelj** [comment]
Aside from the discussion of the plan for the overall icon library (which I believe we should get the decision makers in a room to resolve the plan for the library) I have taken time to resolve the icons to a place where flapjack is happy with them. 

In an effort to try and wrap the development of these up I pass them on to Design Ops. Below is a comparison of the adjusted icons (left column) to the icons that would have been passed on to Lee 4/28 from Daniel (right column).

#### Please click through to the image to see the icons at their full size comparison is between 35px and 16px. If these are not ready to be considered final please let us know of the next actionable step we can take to get them done.

![fj_blog_minicon_adj](https://cloud.githubusercontent.com/assets/6873734/2833646/7a9ee9d8-cfd3-11e3-877c-4b2c960236e6.png)

**Comment by elizbond** [comment]
@juidai @duelj @ell0ell0 I second @Dnpizarro, we understand and intend to work with you guys as these issues come along but after reviewing the newsroom list we saw potential opportunities for us to merge minicons as some tend to overlap. This would be helpful to identify across the board. And there's still the question of do we want to merge and overlap, or should every icon be unique? This obviously changes the scope of our involvement quite a bit when there are other projects we are all working on. I'm trying to balance our project capacity with our involvement here so understanding the full scope is beneficial to us as a team.

@juidai from a UX perspective, do you think reusing the same icon in a different location (like the megaphone being in both newsroom and the blog) could create confusion? Are the icons ever going to be featured in the same feed, like on the home page?

I think @nataliafitzgerald and I are leaning more towards the "each icon should have a unique meaning" approach. If the megaphone is used in one place to mean one thing it should mean the same thing regardless of where it is on the site. However, we currently don't have a full understanding of where icons will be used.

If we decide to go with this approach, we may want to revisit this list of icons since something here might be a better fit on the newsroom list. As different sets come up, this should always be a part of the process.

To finish the current list, I think we need a clearer understanding of the use cases. The "engagement & events" description provided by @juidai is different than the original ask. Here is the current state of each icon, I have noted where we need clarification--
- **share** - _How does everyone feel about the option provided by @duelj?_
- **blog** - _what's the use case here since these are all blog posts?_
- **consumer information** -  blogs that is consumer centric i.e. paying for college, tips on purchasing a home - _latest version provided by @duelj, is this all set?_
- **engagements & events** - "Engagements and Events" will comprise of blog posts announcing CFPB's field hearings, ask users to engage us about the rules we are in the process of creating, RFI (request for information) on a topic such as student credit cards, Know before you owe - consumer feedback on mortgage forms (http://www.consumerfinance.gov/blog/prepaid-cards-help-design-a-new-disclosure/), etc. It's a forum to ask consumer and industry folks to engage with us. 
- **announcements & updates** - regulation type of news or issues the agency will try to tackle - _is this just regulation news, or is it also consumer announcements? Does the megaphone by @duelj still work?_
- **innovation** - blog posts around e-regs, hmda, or anything that is unique and interesting cfpb is doing that is out of the box innovative stuff -  _latest version provided by @duelj, is this all set?_

**Comment by nataliafitzgerald** [comment]
@duelj Will the most common use case for these be 16 pixels? Is 20 pixels a possibility as well? As I look these over, I just want to be sure that these work well at the most common size they will be used.

**Comment by juidai** [comment]
some blog posts will be in newsroom as well. it will carry the blog icon. there might be other areas as well, but as of right now, that's what i know. 

@duelj posts is what we agreed upon and like.

**Comment by duelj** [comment]
@nataliafitzgerald we believe these should be built to work inline with body copy so they should work at 16px. This is kinda the worst case scenario and I assume that if they work this small scaling up should not be a problem. I don't think that the icons in the black circles should be used this small but showed them as a point of reference for some of the counter adjustments. 

#### I do acknowledge that there are some alignment adjustments that need to be done for icons in circles (such as the blog)

**Comment by ell0ell0** [comment]
@nataliafitzgerald anywhere from our paragraph size (16pt) on up will be a possible use. (18pt) will probably be the most common use size, but the minicons will get used inline with text for links, downloads, buttons, etc. a lot.

**Comment by elizbond** [comment]
@juidai we had a lengthy discussion about both the newsroom and blog sets this morning and we just want to make sure we are understanding their relationships. Based on your description, we are assuming the blog posts feed into Newsroom, but instead of having category icons (consumer information, events and engagements, innovation, and announcements) the posts will have the label "blog post" with a more general icon. Is this correct?

Here's an outline of how we are interpreting this (please ignore my scribble)--

![image](https://cloud.githubusercontent.com/assets/6798337/2844271/b911ed40-d088-11e3-884d-8db83611d30b.jpeg)

###### Share will be used throughout the site and is not a category or a label, but an action so it should be a separate issue.

#### Questions
- Is there ever a circumstance where the blog categories and the newsroom labels will live together?
- Are the icons ever going to be featured in the same feed, like on the home page?

#### Daniel will be posting updates shortly.

**Comment by juidai** [comment]
@elizbond - the drawing is correct. newsroom will republish some blog content. it sounds confusing at first but their users will come to newsroom and not necessarily go to blog. some blog content will be news worthy so they want it to be republished in newsroom. a generic blog icon will be easier since their users are not filtering newsroom by the same categories as blogs. 

#### Questions
- I don't think there is a case for blog categories and newsroom labels to live together but I also can't rule it out. As of right now, there's no reason for them to live together. However, we will also need a newsroom minicon because further down the line, we will have a site feed (think Facebook) that will have all our dynamic content show up together. When this happens, blog content (using blog minicon) and news content (using newsroom minicon) will appear on the same page.
- I think you are asking if newsroom categories and newsroom categories will ever be placed next to each other. I stated before, as far as I can see, it is no. But I can't rule it out completely because there might be a use-case we have not considered.

**Comment by elizbond** [comment]
@juidai - Great, thanks for clarifying. I'll update #166 accordingly so we can move to that next.

**Comment by elizbond** [comment]
The share icon has been moved to issue #172.

The Newsroom minicon (for the platform, not its categories) has moved to #171.

**Comment by Dnpizarro** [comment]
Here are our recommendations for the blog minicons (minus "Events & engagements):

![screen shot 2014-04-30 at 6 21 25 pm](https://cloud.githubusercontent.com/assets/6852363/2847961/26e6b954-d0b6-11e3-88d4-a9c93efc27e8.png)

**Comment by duelj** [comment]
Here are some concepts I had for events and engagements maybe they will help the thought process. As with the op-ed newsroom icon just wanted to make sure that you weren't waiting on us for something regarding the events & engagements icon. 
![screen shot 2014-05-01 at 11 45 05 am](https://cloud.githubusercontent.com/assets/6873734/2853779/79959a3e-d148-11e3-9c62-73c84b7cb4c1.png)

**Comment by elizbond** [comment]
@duelj I think that's an interesting concept but it goes back to thinking of "Events and Engagements" as an arrangement to do something or go somewhere at a fixed time, not engage or interact with the CFPB. It needs to be something that communicates receiving information or having a dialogue. With that said, I'm totally stumped on what that should be.

**Comment by elizbond** [comment]
@duelj ooh! What about the interlocking dialogue option from #166 (cc @Scotchester )? Does something like that work or is it too vague for "Events and Engagement"?

**Comment by duelj** [comment]
I like that idea, and so did the team. We chatted about what we thought it might need and we liked the more directional speech bubble from the blog icon so I tried that and a version with the existing bubble with the points shifted. Our intent here was we  felt like the offset points made it feel more like an interaction or conversation. Also, the filled and stroked difference made it feel more like the comments were coming from different people which strengthened that engagement idea.

What are everyone else's thoughts
![screen shot 2014-05-01 at 4 49 22 pm](https://cloud.githubusercontent.com/assets/6873734/2857052/a8ec28be-d172-11e3-8b13-dd7843f3d993.png)
?

**Comment by ell0ell0** [comment]
I like it. Here's a slight refinement of the option on the left. Slightly larger triangle dialogue piece, a little more overlap o speech bubbles, slightly wider speech bubbles, makes it a little more square so it's sits a bit better in the circle.

![engagement](https://cloud.githubusercontent.com/assets/3589375/2864660/1ae4e0d2-d213-11e3-9dde-ae6b879303bf.png)

**Comment by Dnpizarro** [comment]
I like the direction for "Events and Engagements". I had a sketch from some earlier iterations with the 2 bubble concept. I thought I would share anyways and get your thoughts on it.

![screen shot 2014-05-02 at 2 53 30 pm](https://cloud.githubusercontent.com/assets/6852363/2867152/2b02106a-d232-11e3-8b59-1f3d2f94847e.png)

In addition to that, we've been consider just having the 1 bubble option as the "Blog" minicon.

![screen shot 2014-05-02 at 3 40 44 pm](https://cloud.githubusercontent.com/assets/6852363/2867162/54509e3c-d232-11e3-8648-b712cc76358d.png)

I know there are lots of bubble concepts being considered, so it would be useful to be selective about where we decide to make use of it.

**Comment by elizbond** [comment]
I prefer the overlap by @Dnpizarro it is fitting better within the circle.

**Comment by nataliafitzgerald** [comment]
@Dnpizarro can you post a version of the "Events and engagements" without the lines? Can you show an option with two white bubbles and an option with one black one white?

**Comment by nataliafitzgerald** [comment]
@Dnpizarro - For blog the lines on the quote bubble are interesting as they suggested writing.

**Comment by Dnpizarro** [comment]
Here are the revised minicons based on @nataliafitzgerald suggestions:

![screen shot 2014-05-02 at 6 21 38 pm](https://cloud.githubusercontent.com/assets/6852363/2868520/aa6b5fe4-d248-11e3-8734-bfe0f651fece.png)

**Comment by duelj** [comment]
FJ team discussed where things stand with these icons and we like the "events & engagements" icon with one bubble filled and one stroked. In the circle we are cool with them both being filled. 

#### We like all of the other blog category icons as they are.

##### Blog icon:

We have some concerns with he blog icon we feel that the empty bubble could work as a last resort but we still really like the pencil and bubble icon. Where do we stand on being able to use this one?

**Comment by elizbond** [comment]
@duelj I agree with the events and engagement icon.

**Comment by elizbond** [comment]
Closing this, resolved in #166
